src/responsive-breakpoints/responsive-breakpoints.js

```diff
@@ -0,0 +1,22 @@
+window.addEventListener("resize", setBodyCssClasses);
+
+setBodyCssClasses();
+
+function setBodyCssClasses() {
```

Comment:
@joeldenning I haven't seen this pattern (< breakpoint and > breakpoint) before, I've always seen it just "< breakpoint". What's the inspiration/idea?